src/gt4py/next/program_processors/runners/dace_common/workflow.py

```diff
@@ -25,22 +26,20 @@
 
 class CompiledDaceProgram(stages.CompiledProgram):
     sdfg_program: dace.CompiledSDFG
-    # Map SDFG argument to its position in program ABI; scalar arguments that are not used in the SDFG will not be present.
+    # Map SDFG positional arguments to the position in program ABI;
+    # scalar arguments that are not used in the SDFG will not be present.
     sdfg_arg_position: list[Optional[int]]
+    # Map arguments for connectivity tables to the position in program ABI;
+    # consider only the connectivity arrays, skip shape and stride symbols.
+    sdfg_conn_position: dict[str, int]
```

Comment:
Are you still needing these?
According to my understanding everything is inside `inp.sdfg_arglist`.